packages/mux-audio/src/index.ts

```diff
@@ -304,7 +328,7 @@ class MuxAudioElement extends CustomAudioElement<HTMLAudioElement> implements Pa
 
 type MuxAudioElementType = typeof MuxAudioElement;
 declare global {
-  var MuxAudioElement: MuxAudioElementType;
+  var MuxAudioElement: MuxAudioElementType; // eslint-disable-line
```

Comment:
question(non-blocking): What were the linting errors here?

Reply:
the use of `var`. eslint doesn't know this is typescript, I know it's kind of silly 

they even ignored this valid issue
https://github.com/typescript-eslint/typescript-eslint/issues/3617